salt/transport/tcp.py

```diff
@@ -556,6 +557,11 @@ def close(self):
                     raise exc
             self._socket.close()
             self._socket = None
+        if hasattr(self.req_server, 'stop'):
+            try:
+                self.req_server.stop()
+            except Exception as exc:
+                log.debug("TCPReqServerChannel close generated an exception: %s", str(exc))
```

Comment:
I think `log.exception` would be better here.

Reply:
I think I can get rid of that completely by checking is_alive in the `stop` method. If not, I'll change this to log exception.